index.html

```diff
@@ -150,5 +150,36 @@ <h3 class="framework-text">Skills</h3>
                 <li>Terminal</li>
             </ul>
         </section>
+        <section>
+            <p class="text-form">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form>
+                <label for = "name"></label>
+                <input type="text" id="name" class="input-padron" required placeholder="Full name">
+                <hr>
+                
+                <label for="mail"></label>
+                <input type="email" id="mail" class="input-padron" required placeholder="email address">
+                <hr>
+    
+                <label for="msj"></label>
+                <textarea rows="4" cols="50" maxlength="500" id="msj" placeholder="Enter text here"></textarea>
+                <button class="form-buttom" type="submit">Get in touch</button>
+            </form>
+            <hr>
```

Comment:
- [ ] Good job implementing the hover and pressed state with `:hover` and `:active` pseudoclasses but the issue is your colors on button interactions are not following the figma design. kindly consider changing the hover and pressed state with following Github button interactions guide. and also dont forget to apply these button interactions styling in all of your buttons on the webpage.
figma design guidelines
![qw](https://user-images.githubusercontent.com/96683843/221255129-e52164fb-2ab7-4eeb-91ea-7faf90320e48.jpg)
you may want to focus on hover and pressed button color styling